lib/cpp/hawkes/simulation/simu_hawkes.cpp

```diff
@@ -56,15 +53,13 @@ bool Hawkes::update_time_shift_(double delay, ArrayDouble &intensity,
 void Hawkes::reset() {
   for (unsigned int i = 0; i < n_nodes; i++) {
```

Comment:
I know it was `unsigned int` before, I would generally prefer `std::uint32_t` or similar